awx/ui_next/src/screens/Setting/shared/SharedFields.jsx

```diff
@@ -76,6 +77,32 @@ const SettingGroup = withI18n()(
   )
 );
 
+const ExecutionEnvironmentLookupField = withI18n()(({ i18n }) => {
```

Comment:
Since this lookup is used in only one place (MiscSystemEdit) within Settings, it probably doesn't need to be in SharedFields and instead we could import the lookup directly from /components in MiscSystemEdit.